test/powershell/Modules/Microsoft.PowerShell.Security/ExecutionPolicy.Tests.ps1

```diff
@@ -998,9 +1003,43 @@ ZoneId=$FileType
             Set-ExecutionPolicy -Scope CurrentUser -ExecutionPolicy ByPass
             Get-ExecutionPolicy -Scope CurrentUser | Should Be "ByPass"
         }
+    }
+
+    Describe "Validate Set-ExecutionPolicy -Scope (Admin)" -Tags @('CI', 'RequireAdminOnWindows') {
+
+        BeforeAll {
+            if ($IsNotSkipped)
+            {
+                $originalPolicies = Get-ExecutionPolicy -list
+            }
+        }
+
+        AfterAll {
+            if ($IsNotSkipped)
+            {
+                RestoreExecutionPolicy $originalPolicies
+            }
+        }
+
+        It '-Scope LocalMachine is Settable, but overridden' {
+            # setup
+            Set-ExecutionPolicy -Scope Process -ExecutionPolicy Undefined
+            Set-ExecutionPolicy -Scope CurrentUser -ExecutionPolicy Restricted
+            $errorId = $null
+            try
+            {
+                Set-ExecutionPolicy -Scope LocalMachine -ExecutionPolicy ByPass
+            }
+            catch [System.Security.SecurityException] {
+                $errorId = $_.FullyQualifiedErrorId
+            }
+
+            $errorId | Should Be 'ExecutionPolicyOverride,Microsoft.PowerShell.Commands.SetExecutionPolicyCommand'
+            Get-ExecutionPolicy -Scope LocalMachine | Should Be "ByPass"
```

Comment:
Nope, that's how PowerShell works. The full error message text is:
```

Set-ExecutionPolicy : Windows PowerShell updated your execution policy successfully, but the setting is overridden by
a policy defined at a more specific scope.  Due to the override, your shell will retain its current effective
execution policy of RemoteSigned. Type "Get-ExecutionPolicy -List" to view your execution policy settings. For more
information please see "Get-Help Set-ExecutionPolicy".

FullyQualifiedErrorId : ExecutionPolicyOverride,Microsoft.PowerShell.Commands.SetExecutionPolicyCommand
```

**Windows PowerShell updated your execution policy successfully, but the setting is overridden by
a policy defined at a more specific scope.**